rules-reviewed/eap7/eap6/hibernate4-xml.windup.xml

```diff
@@ -11,8 +11,10 @@
             <addon id="org.jboss.windup.rules,windup-rules-java,2.4.0.Final" />
         </dependencies>
         <sourceTechnology id="hibernate" versionRange="[4,5)" />
+        <sourceTechnology id="eap" versionRange="[6,7)" />
         <targetTechnology id="hibernate5" versionRange="[5,)" />
         <targetTechnology id="eap7" versionRange="[7,)" />
+        <targetTechnology id="eap" versionRange="[7,)" />
```

Comment:
Is this OK? I don't think we've been using target technology of `eap` for EAP 7, only `eap7`.

I mean, if we were able to properly use the version ranges when invoking Windup (like `windup --source eap:6 --target eap:7`), that would be awesome, but we're not there yet, no?